Library/DataSource/UIView-Extensions.swift

```diff
@@ -25,6 +25,8 @@ private func swizzle(_ v: UIView.Type) {
 }
 
 extension UIView {
+  // FIXME: "Method 'initialize' defines Objective-C class method, which is not guaranteedto be invoked by 
+  // by Swift and will be disallowed in future versions.
   open override class func initialize() {
```

Comment:
gotta figure this one out before merging :/ still thinking about possible solutions